src/test/fuzz/validation_load_mempool.cpp

```diff
@@ -36,9 +37,12 @@ FUZZ_TARGET_INIT(validation_load_mempool, initialize_validation_load_mempool)
 
     CTxMemPool pool{MemPoolOptionsForTest(g_setup->m_node)};
 
+    auto& chainstate{static_cast<DummyChainState&>(g_setup->m_node.chainman->ActiveChainstate())};
+    chainstate.SetMempool(&pool);
+
     auto fuzzed_fopen = [&](const fs::path&, const char*) {
         return fuzzed_file_provider.open();
     };
-    (void)LoadMempool(pool, g_setup->m_node.chainman->ActiveChainstate(), fuzzed_fopen);
+    (void)chainstate.LoadMempool(MempoolPath(g_setup->m_args), fuzzed_fopen);
```

Comment:
In commit "LoadMempool: Pass in load_path, stop using gArgs" (678036771720433bd789e68e43c0e250cd3706b8)

What's the reason this fuzz test is changed to use chainstate.LoadMempool instead of kernel::LoadMempool? It seems to make test setup more fragile and LoadMempool/DumpMempool sequence a little less symmetric. Maybe extra test coverage is worth it though or there was a different reason previous code didn't work. Just curious

Reply:
Added more context in c4249c1243981f42091950d1295c2be6bd4bf118, also added a followup to the project ticket to make loadmempool less footgunny (this footgun was existed prior to this PR)